src/app/commands/cmd_new_layer.cpp

```diff
@@ -227,6 +232,11 @@ void NewLayerCommand::onExecute(Context* context)
     if (window.closer() != window.ok())
       return;
 
+    pref.tileset.gridWidth(tilesetSelector->getInfo().grid.tileSize().w);
+    pref.tileset.gridHeight(tilesetSelector->getInfo().grid.tileSize().h);
+    pref.tileset.baseIndex(tilesetSelector->getInfo().baseIndex);
+    pref.tileset.save();
```

Comment:
There is no need to call save() immediately here (`Preferences::save()` is called in `App::~App()` anyway and is the real function that saves to disk -or should be-)
```suggestion
```